core/include/cib.h

```diff
@@ -18,8 +18,10 @@
  * @author      unknown, propably Kaspar Schleiser <kaspar@schleiser.de>
  */
 
-#ifndef __CIB_H
-#define __CIB_H
+#include <stdint.h>
+
+#ifndef CIB_H
+#define CIB_H
```

Comment:
I took the liberty to correct that too even if this isn't the main concern of this PR.